taxi/tests/test_views.py

```diff
@@ -0,0 +1,218 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car
+
+INDEX_URL = reverse("taxi:index")
+MANUFACTURER_URL = reverse("taxi:manufacturer-list")
+CAR_URL = reverse("taxi:car-list")
+DRIVER_URL = reverse("taxi:driver-list")
+CAR_DETAIL_URL = reverse("taxi:car-detail", args=[1])
+
+
+class PublicIndexTest(TestCase):
+    def test_loggin_required(self):
+        res = self.client.get(INDEX_URL)
+        self.assertNotEqual(res.status_code, 200)
+
+
+class PrivateIndexTest(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="test",
+            password="12345",
+            license_number="TTT12345"
+        )
+        self.client.force_login(self.user)
+
+    def test_retrive_index(self):
+        res = self.client.get(INDEX_URL)
+        self.assertTrue(res.status_code, 200)

```

Comment:
The assertion `self.assertTrue(res.status_code, 200)` is incorrect. Use `self.assertEqual(res.status_code, 200)` to properly check the status code.